src/diagram/components/dialog/number-row.tsx

```diff
@@ -1,28 +1,35 @@
 import React from "react";
-
 import { DialogRow } from "./dialog-row";
 import { NumberInput } from "../ui/number-input";
 import { isValidNumber } from "../../utils/validate";
+import classNames from "classnames";
```

Comment:
Move import ⬆️ to keep external imports together

Reply:
@kswenson @tealefristoe Thanks. I didn't see the pattern, but get it now.

I've left in or added the separating newline in the latest commit. I'm kind of ambivalent about the new line, too, but I don't mind it now that I know the purpose. We have a somewhat similar pattern with separating the stylesheet imports with a newline. Seems fine to me.